### ISSUE TITLE
Mine with specific devices.

### ISSUE THREAD

**Issue by okumedia** [bzminer/bzminer#179]
How do I select devices to minee with? e.g --device 1,2,4 etc did nothing.

**Comment by Jorgelote** [comment]
Windows? just double click on "from config.bat" and then go to config.txt and change "true" to "false" on the devices you do not want, that will disable all devices selected as "false"

**Comment by Stey-U** [comment]
Yep I was doing the same for ages (in Windows), then realised I can also put the command --disable 1:0,2:0,3:0 etc for the GPUs you don't want to mine on in the .bat file. Either method works fine.
